test/parallel/test-dotenv-edge-cases.js

```diff
@@ -35,4 +37,17 @@ describe('.env supports edge cases', () => {
     assert.strictEqual(child.code, 0);
   });
 
+  it('should support absolute paths', async () => {
+    const code = `
+      require('assert').strictEqual(process.env.CUSTOM_VARIABLE, 'hello-world');
+    `.trim();
+    const child = await common.spawnPromisified(
+      process.execPath,
+      [ `--env-file=${absolutePath}`, '--eval', code ],
```

Comment:
You don't know if the absolute path contains special char, it's more prudent to pass it as a separate argument.
```suggestion
      [ `--env-file`, absolutePath, '--eval', code ],
```

Reply:
Nice catch, makes sense! Thank you for the review @aduh95